include/sidekiq/sidekiq_rx.h

```diff
@@ -38,6 +38,9 @@ namespace gr {
 			typedef std::shared_ptr<sidekiq_rx> sptr;
 
 			static sptr make(
+                    int _card,
```

Comment:
suggest changing _card to card to match other variable naming convention

Reply:
ok

Reply:
done